### PR TITLE
Dynamic Cart Total for Navigation

### PR DESCRIPTION
**Pull request by ghost**
Within this pull request you will find that whenever a user adds an album to
their cart that it will display as so on the nav bar, "Cart (albums_quantity)".
I also made sure to add the logic necessary to make sure if a user doesn't have
anything in their cart that it will not display "Cart (0)" but rather "Cart".

closes #55 
@julsfelic @Draganovic @scottfirestone 

Hey guys, check out the code and let me know what you think!

### CODE REVIEW

**Comment by Salvi6God** [comment]
I'll fix these hound issues asap!

File: spec/features/guest_can_view_total_amount_of_items_in_their_cart_on_the_navigation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.feature "Guest sees total amount of items in their cart on the navigation" do
+  scenario "guest can see total items on navbar" do
+    album = FactoryGirl.create(:album)
```
Comment (julsfelic) [inline spec/features/guest_can_view_total_amount_of_items_in_their_cart_on_the_navigation_spec.rb]: Go ahead and get rid of the variable assignment on line 5. All you need is `FactoryGirl.create(:album)`

File: app/models/cart.rb [hunk 2]
```diff
@@ -57,4 +57,10 @@ def add_one(album_id)
   def minus_one(album_id)
     @contents[album_id.to_s] -= 1
   end.to_s
+
+  def navigation_quantity
+    if quantity != 0
```
Comment (julsfelic) [inline app/models/cart.rb]: You could clean up this code a little like this:

``` ruby
  def navigation_quantity
    "( #{quantity} )" unless quantity.zero?
  end
```

**Comment by julsfelic** [comment]
Left some comments. Minor suggestions but overall looks good. I'll merge it in once the changes are made.

**Comment by ghost** [comment]
fixed the issues that hound and @julsfelic pointed out!

**Comment by scottfirestone** [comment]
Looks good! Love how you have "cart (0)" just show "cart". Merging...

**Pull request merged**